index.html

```diff
@@ -114,6 +114,96 @@ <h3 class="skills">Skills</h3>
             </li>
          </ul>
       </div>
-   </section> 
+   </section>
+
+   <section class="recent-work">
+      <h1 class="rw">MY RECENT WORK</h1>
+      <div class="divider"></div>
+      <div class="work1">
+         <div class="img1">
+            <img src="images/img.png" class="image" alt="software-portfolio">
+         </div>
+         <div class="info1">
+            <h3 class="sub1">Multi-Post Stories</h3>
+            <div class="circle3"></div>
+            <p class="desc1">
+               A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <ul class="main-works">
+               <li class="works">css</li>
+               <li class="works">html</li>
+               <li class="works">bootstrap</li>
+               <li class="works none">Ruby</li>
+            </ul>
+            <div class="button">
+               <a href="" class="main-button">See Project</a>
+            </div>
+         </div>
+      </div>
+      <div class="work1">

```

Comment:
- [ ] Another requirement is to use CSS flexbox to align the content inside the work card. So please make sure to implement the necessary changes to solve this issue.